src/cares_wrap.cc

```diff
@@ -1063,17 +1166,14 @@ int ParseSoaReply(Environment* env,
 
 class QueryAnyWrap: public QueryWrap {
  public:
-  QueryAnyWrap(Environment* env, Local<Object> req_wrap_obj)
-    : QueryWrap(env, req_wrap_obj) {
+  QueryAnyWrap(ChannelWrap* channel, Local<Object> req_wrap_obj)
+    : QueryWrap(channel, req_wrap_obj) {
   }
 
   int Send(const char* name) override {
-    ares_query(env()->cares_channel(),
-               name,
-               ns_c_in,
-               ns_t_any,
-               Callback,
-               GetQueryArg());
+    AresQuery(name,
+              ns_c_in,
+              ns_t_any);
```

Comment:
This could as well be on one line IMO to be consistent with `QueryAWrap::Send` etc.